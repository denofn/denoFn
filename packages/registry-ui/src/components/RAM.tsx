import React from "react";
import { css } from "otion";

export function RAM({ children }: React.PropsWithChildren<{}>) {
  return (
    <div
      className={css({
        width: "100%",
        display: "grid",
        gridGap: "1rem",
        gridTemplateColumns: "repeat(auto-fill, minmax(252px, 1fr))",
      })}
    >
      {children}
    </div>
  );
}
