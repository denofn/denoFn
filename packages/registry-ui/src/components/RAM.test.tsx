import { render, cleanup } from "@testing-library/react";
import React from "react";

import { RAM } from "./RAM";

afterEach(cleanup);

it("renders with the correct styles", () => {
  const { asFragment } = render(<RAM />);
  expect(asFragment()).toMatchSnapshot();
});
