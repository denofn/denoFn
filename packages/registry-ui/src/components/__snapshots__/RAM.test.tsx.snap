// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
<DocumentFragment>
  <div
    class="_1d3w5wq _vetbs0 _197uonj _brfjq5"
  />
</DocumentFragment>
`;
